kuma/landing/views.py

```diff
@@ -33,6 +33,7 @@ def home(request):
         'updates': updates,
         'stats': community_stats,
         'current_challenge_tag_name': devderby_tag,
+        'template_debug': settings.TEMPLATE_DEBUG,
```

Comment:
No need to pass that into the template, the `settings` object is passed to all template contexts automatically via the `global_settings` template context processor (https://github.com/mozilla/kuma/blob/ae6c3d233c3503994f0d30671108b6d0d756effd/kuma/core/context_processors.py#L5-L7). Just refer to the setting in question as ``settings.TEMPLATE_DEBUG``.